<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-F1663RVI" name="com.HerkoappTests.ValidateAlertPopUpTest" tests="8" failures="1" timestamp="9 Sep 2020 00:48:59 GMT" time="12.989" errors="0">
  <testcase name="clickOnShwAlertBtn" time="3.138" classname="com.HerkoappTests.ValidateAlertPopUpTest"/>
  <testcase name="handleAlert" time="2.025" classname="com.HerkoappTests.ValidateAlertPopUpTest"/>
  <testcase name="launchWebSite" time="3.096" classname="com.HerkoappTests.ValidateAlertPopUpTest"/>
  <testcase name="clickOnPromptAlert" time="4.129" classname="com.HerkoappTests.ValidateAlertPopUpTest"/>
  <testcase name="validateWebsite" time="0.033" classname="com.HerkoappTests.ValidateAlertPopUpTest"/>
  <testcase name="validateAlertPage" time="0.015" classname="com.HerkoappTests.ValidateAlertPopUpTest"/>
  <testcase name="clickOnAlertJsLnk" time="0.512" classname="com.HerkoappTests.ValidateAlertPopUpTest"/>
  <testcase name="validatePromptAlertValue" time="0.041" classname="com.HerkoappTests.ValidateAlertPopUpTest">
    <failure type="java.lang.AssertionError" message="expected [Hari kumar selenium] but found [Selenium]">
      <![CDATA[java.lang.AssertionError: expected [Hari kumar selenium] but found [Selenium]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.HerkoappTests.ValidateAlertPopUpTest.validatePromptAlertValue(ValidateAlertPopUpTest.java:100)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- validatePromptAlertValue -->
</testsuite> <!-- com.HerkoappTests.ValidateAlertPopUpTest -->
